includes/class-newspack-popups-inserter.php

```diff
@@ -70,7 +70,7 @@ public function __construct() {
 	 * @return string The content with popup inserted.
 	 */
 	public static function popup( $content = '' ) {
-		if ( is_admin() ) {
+		if ( is_admin() || is_feed() ) {
```

Comment:
Is this hook only supposed to run for popups? If so, I'd recommend making this conditional check that the current post is of the right post type. E.g.:

```
if ( Newspack_Popups::NEWSPACK_PLUGINS_CPT !== get_post_type() ) {
    return;
}
```

Reply:
No, it injects the popup into post content.